webofneeds/won-bot/src/main/java/won/bot/framework/eventbot/action/impl/atomlifecycle/AbstractActivateAtomAction.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2012 Research Studios Austria Forschungsges.m.b.H. Licensed under
+ * the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable
+ * law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package won.bot.framework.eventbot.action.impl.atomlifecycle;
+
+import org.apache.jena.query.Dataset;
+import won.bot.framework.eventbot.EventListenerContext;
+import won.bot.framework.eventbot.action.BaseEventBotAction;
+import won.protocol.message.WonMessage;
+import won.protocol.message.WonMessageBuilder;
+import won.protocol.util.WonRdfUtils;
+
+import java.net.URI;
+
+/**
+ * Base class for actions that deactivates atoms.
+ */
+public abstract class AbstractActivateAtomAction extends BaseEventBotAction {
```

Comment:
Javadoc copy paste mistakes, the comments are about deactivating atoms, the class activates atoms